### ISSUE TITLE
`pkg` folder isn't included in the published npm package

### ISSUE THREAD

**Issue by kevinbarabash** [wasmerio/wasmer-js#301]
This causes issues with the typescript definitions since `dist/wasi.d.ts` contains the following:
```
export * from "./pkg/wasmer_wasi_js";
import { InitInput } from "./pkg/wasmer_wasi_js";
export declare const init: (input?: InitInput | Promise<InitInput>, force?: boolean) => Promise<void>;
```

**Comment by syrusakbary** [comment]
This is now fixed in published npm versin `1.1.1`

**Comment by kevinbarabash** [comment]
@syrusakbary awesome!  Thanks for addressing this so quickly. 🙂
